tests/api-diff/run.t

```diff
@@ -28,3 +28,44 @@ there should be no diff:
 
   $ api-diff ref.cmi ref.cmi
   API unchanged!
+
+# Adding a module
+  $ cat > mod_add.mli << EOF
+  > module Hello :
```

Comment:
Here you're not simply adding a module compared to `ref.mli`, you're also removing the type and value that is there.

The test case should showcase a single change between the two `.cmi` files.